xarray/core/dataset.py

```diff
@@ -2486,6 +2486,35 @@ def isel(
             in this dataset, unless vectorized indexing was triggered by using
             an array indexer, in which case the data will be a copy.
 
+        Example
+        -------
+
+        import xarray as xr
+
+        dataset = xr.Dataset(
+            {
+                'math_scores': (
+                    ['student', 'test'],
+                    [[90, 85, 92],
+                    [78, 80, 85],
+                    [95, 92, 98]]
+                ),
+                'english_scores': (
+                    ['student', 'test'],
+                    [[88, 90, 92],
+                    [75, 82, 79],
+                    [93, 96, 91]]
+                )
+            },
+            coords={
+                'student': ['Alice', 'Bob', 'Charlie'],
+                'test': ['Test 1', 'Test 2', 'Test 3']
+            }
+        )
+
+        second_student_first_test = dataset.isel(student=1, test=0)
+        print(second_student_first_test)
```

Comment:
```suggestion
        second_student_first_test
```
Then paste the output of calling this in ipython on the line below as we discussed